core-backend/common/src/funcs.rs

```diff
@@ -200,9 +200,21 @@ where
         Err(ActorTerminationReason::Exit(inheritor_id).into())
     }
 
-    #[host(fallible, cost = RuntimeCosts::StatusCode, err_len = LengthWithCode)]
-    pub fn status_code(ctx: &mut R) -> Result<(), R::Error> {
-        ctx.ext_mut().status_code().map_err(Into::into)
+    #[host(fallible, cost = RuntimeCosts::ReplyCode, err_len = LengthWithReplyCode)]
+    pub fn reply_code(ctx: &mut R) -> Result<(), R::Error> {
+        ctx.ext_mut()
+            .reply_code()
+            .map(ReplyCode::to_bytes)
+            .map_err(Into::into)
+    }
+
+    // TODO (breathx): change cost.
```

Comment:
add an issue if it is still TODO

Reply:
Yep. Each TODO with my nickname is tmp comment. Already resolved and opened issue. To be pushed once finished the very last part of pr